Templates/GitHubCommenter-FSharp/run.fsx

```diff
@@ -27,24 +29,61 @@ let SendGitHubRequest (url: string) requestBody =
         return! client.PostAsync(url, content) |> Async.AwaitTask
     } |> Async.RunSynchronously
 
-let Run(payload: obj, log: TraceWriter) =
+let rec hasProp (key: string list) (from: JObject) =
+    match from with
+    | null -> false
+    | _ ->
+        let x = from.[key.Head]
+        match x with
+        | null -> false
+        | _ ->
+            match key with
+            | [_] -> true
+            | _::tl -> hasProp tl (x.Value<JObject>())
+            | [] -> false
+
+let rec prop<'T> (key: string list) (def: 'T) (from: JObject) =
+    match from with
+    | null -> def
+    | _ ->
+        let x = from.[key.Head]
+        match x with
```

Comment:
Normally this would be done with `FSharp.Interop.Dynamic`, but unfortunately it fails to run on Azure Functions. When that's solved, this can be (drastically) simplified.